### PR TITLE
Pbi 620 default organ rotation

### PR DESCRIPTION
**Pull request by axdanbol**

### CODE REVIEW

**Comment by axdanbol** [comment]
@bherr2 Should I add any default rotations to the organs? If so which organs and what rotations?

**Comment by bherr2** [comment]
> @bherr2 Should I add any default rotations to the organs? If so which organs and what rotations?

@lecross should be getting you exact rotations. Heart will be rotated so its facing downard and Kidney should be rotated so that the bisection plain is perpendicular to the starting viewport. In the interim, just put dummy ones in for now to demonstrate different variations.

File: projects/ccf-shared/src/lib/components/organ-selector/organ-selector.component.ts [hunk 1]
```diff
@@ -1,37 +1,69 @@
...
+    id: 'http://purl.obolibrary.org/obo/UBERON_0002097'
+  },
+  {
+    src: 'app:large_intestine', organ: 'Large Intestine', name: 'Large Intestine', hasSex: true,
+    id: 'http://purl.obolibrary.org/obo/UBERON_0000059'
+  },
+  {
+    src: 'app:heart', organ: 'Heart', name: 'Heart', hasSex: true,
+    id: 'http://purl.obolibrary.org/obo/UBERON_0000948',
+    defaultTransform: new Matrix4(Matrix4.IDENTITY).rotateX(Math.PI / 2)
```
Comment (bherr2) [inline projects/ccf-shared/src/lib/components/organ-selector/organ-selector.component.ts:23]: This needs to be [number, number, number].. Apply the matrix elsewhere, I would like to keep this 'data only'. Use the rotateXYZ function as well.

**Comment by bherr2** [comment]
Needs tested more. When I move the tissue block around it acts _really_ wonky (tried with heart).

**Comment by axdanbol** [comment]
Closing in favor of new pr

**Pull request closed without merging**